MaterialX/config.py

```diff
@@ -0,0 +1,137 @@
+
+			####################################################
+			# UNCOMMENT TO USE MATERIALX PACKAGED PYBIND11     #
+			####################################################
+			" -D MATERIALX_PYTHON_PYBIND11_DIR=source/PyMaterialX/PyBind11"
+			####################################################
+
+			####################################################
+			# UNCOMMENT TO USE GAFFER DEPENDENCIES PYBIND11    #
+			####################################################
+			#" -D CMAKE_MODULE_PATH={buildDir}/share/cmake/pybind11"
+			####################################################
+
+			" -D MATERIALX_BUILD_SHARED_LIBS=TRUE"
+			" -D MATERIALX_BUILD_GEN_MDL=FALSE"
+		
+			" -D MATERIALX_BUILD_PYTHON=TRUE"
+			" -D MATERIALX_PYTHON_EXECUTABLE={buildDir}/bin/python"
+			" -D MATERIALX_PYTHON_OCIO_DIR={buildDir}/openColorIO"
+			" {pythonArguments}"
+			
+			" -D MATERIALX_BUILD_OIIO=TRUE"
+			" -D MATERIALX_OIIO_DIR={buildDir}"
+			" -D MATERIALX_OSL_INCLUDE_PATH={buildDir}/include"
+			" -D MATERIALX_OSLC_EXECUTABLE={buildDir}/bin/oslc"
+			" -D MATERIALX_TEST_RENDER=FALSE"
+			" ."
+		,
+
+		"make install -j {jobs}",
+
+	],
+
+	"variant:Python:2" : { 
+
+		"variables" : { 
+
+			"pythonArguments" : "-D MATERIALX_PYTHON_VERSION=2",
+
+		},
+
+	},  
+
+	"variant:Python:3" : { 
+
+		"variables" : { 
+
+			"pythonArguments" : "-D MATERIALX_PYTHON_VERSION=3",
+
+		},
+
+	},
+
+	"manifest" : [
+
+		"cmake/MaterialXConfig.cmake",
+		"cmake/MaterialXConfig-noconfig.cmake",
+		"cmake/MaterialXConfigVersion.cmake"
+		"lib/libMaterialXCore.so.1.38.0",
+		"lib/libMaterialXCore.so.1",
+		"lib/libMaterialXCore.so",
+		"lib/libMaterialXFormat.so.1.38.0",
+		"lib/libMaterialXFormat.so.1",
+		"lib/libMaterialXFormat.so"
+		"lib/libMaterialXGenShader.so.1.38.0",
+		"lib/libMaterialXGenShader.so.1",
+		"lib/libMaterialXGenShader.so",
+		"lib/libMaterialXGenGlsl.so.1.38.0",
+		"lib/libMaterialXGenGlsl.so.1",
+		"lib/libMaterialXGenGlsl.so",
+		"lib/libMaterialXGenOsl.so.1.38.0",
+		"lib/libMaterialXGenOsl.so.1",
+		"lib/libMaterialXGenOsl.so",
+		"lib/libMaterialXRender.so.1.38.0",
+		"lib/libMaterialXRender.so.1",
+		"lib/libMaterialXRender.so",
+		"lib/libMaterialXRenderOsl.so.1.38.0",
+		"lib/libMaterialXRenderOsl.so.1",
+		"lib/libMaterialXRenderOsl.so",
+		"lib/libMaterialXRenderHw.so.1.38.0",
+		"lib/libMaterialXRenderHw.so.1",
+		"lib/libMaterialXRenderHw.so",
+		"lib/libMaterialXRenderGlsl.so.1.38.0",
+		"lib/libMaterialXRenderGlsl.so.1",
+		"lib/libMaterialXRenderGlsl.so",
+		"include/MaterialXCore",
+		"include/MaterialXFormat",
+		"include/MaterialXGenShader",
+		"include/MaterialXGenGlsl",
+		"include/MaterialXGenOsl",
+		"include/MaterialXRender",
+		"include/MaterialXRenderOsl",
+		"include/MaterialXRenderHw",
+		"include/MaterialXRenderGlsl",
+		"resources/Geometry",
+		"resources/Images",
+		"resources/Lights",
+		"resources/Materials",
+		"libraries/bxdf",
+		"libraries/lights",
+		"libraries/pbrlib",
+		"libraries/stdlib",
```

Comment:
It would be great if we could install these in a subfolder somehow, to make it clear that they're MaterialX things and not Gaffer things. Do you know if that is possible? Also, are the resources perhaps just for use by the viewer (which it seems we're not building)?